### PR TITLE
riscv64: Add RISC-V support

### PR DESCRIPTION
**Pull request by MarekPikula**

### CODE REVIEW

**Comment by MarekPikula** [comment]
How could the RISC-V build be added to TAOS?

**Comment by myungjoo** [comment]
> How could the RISC-V build be added to TAOS?

We will need Tizen-GBS ready for RISC-V and all the dependency packages ready for RISC-V in Tizen, first.

File: packaging/machine-learning-api.spec [hunk 1]
```diff
@@ -24,7 +24,7 @@
 %define		tensorflow2_gpu_delegate_support 0
 %endif
 
-%ifnarch %arm aarch64 x86_64 i586 i686 %ix86
```
Comment (myungjoo) [inline packaging/machine-learning-api.spec:27]: Let's not enable nnfw for RISC-V until nnfw ( https://github.com/Samsung/One ) is ready for RISC-V

Comment (MarekPikula) [reply packaging/machine-learning-api.spec]: Indeed, for now, supporting everything is not our main priority, so I dropped the ifnarch for riscv64.

**Comment by myungjoo** [comment]
@MarekPikula If you want this merged, please mark "Ready for review".

**Review by myungjoo — APPROVED** [review]

**Review by songgot — APPROVED** [review]
LGTM

**Pull request merged**